### DISCUSSION TITLE
Access the UI from any client on a given network prefixe

### DISCUSSION THREAD

**Discussion by NetDevAutomate** [Quansight/ragna#207] [Q&A] [answered]
Firstly, awesome project!!!

When accessing the UI from my local machine all woks really well, however when I try from another laptop I can’t get this to work. 

I’ve tried biding the UI to the IP of the address of the machine running Ragna and also adding this to the origins. 

Although I can access the UI, doc uploads don’t work and depending on config I end up with Tornado errors amongst others. 

Any ideas or sample .toml config?

Many thanks in advance.

**Comment by pmeier** [comment]
Not sure I fully understand the issue. Are the API and UI running on the same machine? Does the ohter machine have access to the machine where the UI is running? Please describe the setup a little more.

Being able to upload documents, but nothing else sounds weird. Usually the document upload is the issue, since for this the browser needs to connect to the API directly. Could you open the developer tools in the browser (press F12) and see if there are any errors when trying to start a chat? I suspect this being a CORS issue.

**Reply by NetDevAutomate** [reply, marked as answer]
My sincere apologies... all fixed.

I needed to set the API and UI with a hard coded address and add that to the origins list on both,
